CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
+        self.assertEqual(7, Calculator.run_operation(7, 10, '%'))
+
     def test_run_operation_invalid(self):
         with self.assertRaises(Exception):
             Calculator.run_operation(9,0, '8')
```

Comment:
This is a good test case to check for an invalid operator. Perhaps you can add another test case if the user inputs another symbol such as '&'.

Reply:
I agree the test case is efficient but it should also take into account other symbols as well

Reply:
I agree as well. Should add test cases for symbols that aren't accepted.